src/BaGet.Aliyun/AliyunOSSService.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Aliyun.OSS;
+using BaGet.Aliyun.Configuration;
+using BaGet.Core;
+using Microsoft.Extensions.Options;
+
+namespace BaGet.Aliyun
+{
+    public class AliyunOSSService : IStorageService
+    {
+        private const string Separator = "/";
+        private readonly string _bucket;
+        private readonly string _prefix;
+        private readonly OssClient _client;
+
+        public AliyunOSSService(IOptionsSnapshot<AliyunOSSOptions> options, OssClient client)
+        {
+            if (options == null)
+                throw new ArgumentNullException(nameof(options));
+
+            _bucket = options.Value.Bucket;
+            _prefix = options.Value.Prefix;
+            _client = client ?? throw new ArgumentNullException(nameof(client));
+
+            if (!string.IsNullOrEmpty(_prefix) && !_prefix.EndsWith(Separator))
+                _prefix += Separator;
+        }
+
+        private string PrepareKey(string path)
+        {
+            return _prefix + path.Replace("\\", Separator);
+        }
+
+        public async Task<Stream> GetAsync(string path, CancellationToken cancellationToken = default)
+        {
+            try
+            {
+                return await Task.Run<Stream>(() =>
```

Comment:
Does the OSS SDK not support async? If so, let's make these methods non-async and return the results using `Task.FromResult(... return value here ...);` instead of using `Task.Run`

Reply:
Thanks for let me know this. 

It does support async but only with IAsyncResult, I didn't have experience to due with IAsyncResult, to make BaGet run in docker (Aliyun Serverless Kubernates) ASAP, I made this in hurry.

I also tested to use AWS options for Aliyun OSS in docker, not work.

Now I make some improvement for async call.